frontend/src/scenes/insights/insightDataLogic.ts

```diff
@@ -242,13 +242,17 @@ export const insightDataLogic = kea<insightDataLogicType>([
             }
         },
         loadInsightSuccess: ({ insight }) => {
-            if (insight.filters) {
+            if (!!insight.query) {
```

Comment:
my gut is this `insightDataLogic` learns to either cope with or completely ignore non insightVisNodes

Reply:
will completely ignore them